lib/ramble/ramble/test/data/reports/results.test.json

```diff
@@ -0,0 +1 @@
+/usr/local/google/home/dpomeroy/VSCode/ramble/var/ramble/workspaces/basic_gromacs/results.2024-06-28_20.55.29.json
```

Comment:
Not a big deal, but do you want such a "real" (and personal) path here?

Reply:
I don't have a strong opinion, I was just trying to keep the data realistic to make sure it's a good test. Is there a test path you'd recommend?

Reply:
Figured it out. Somehow I ended up with a symlink instead of the file inside my workspace. Now fixed